pennylane/measurements/__init__.py

```diff
@@ -24,8 +24,9 @@
 classes. These classes are subclassed to implement measurements in PennyLane.
 
 * Each :class:`SampleMeasurement` subclass represents a sample-based measurement, which contains a
-  :meth:`SampleMeasurement.process_samples` method that processes the sequence of samples generated
-  by the device. This method should always have the same arguments:
+  :meth:`SampleMeasurement.process_samples` and :meth:`SampleMeasurement.process_counts` methods
+  that processes the sequence of samples generated
```

Comment:
```suggestion
  that processe the sequence of samples generated
```

Reply:
updated Matthew's comment with this suggestion.